browser/src/map/handler/Map.FileInserter.js

```diff
@@ -109,12 +109,60 @@ L.Map.FileInserter = L.Handler.extend({
 		this._toInsertBackground = {};
 	},
 
-	_sendFile: function (name, file, type) {
+	_sendFile: async function (name, file, type) {
 		var socket = app.socket;
 		var map = this._map;
 		var sectionContainer = app.sectionContainer;
 		var url = this.getWopiUrl(map);
 
+		var size;
+
+		if (type === 'multimedia') {
+			const videoElement = document.createElement('video');
+			const objectURL = window.URL.createObjectURL(file);
```

Comment:
we need to release memory: https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL_static#memory_management

also there is a note about function beeing deprecated soon: https://developer.mozilla.org/en-US/docs/Web/API/URL/createObjectURL_static#using_object_urls_for_media_streams

Reply:
> we need to release memory

good catch, I'll do that and update the PR...

> also there is a note about function beeing deprecated soon

unluckily, as is often the case with the web, the replacements for the soon-deprecated features are as-of-yet unsuitable: https://caniuse.com/?search=srcObject

luckily, as is often the case with the web, I have doubts about any modern browser removing support for createObjectURL any time soon

Reply:
(on the caniuse page there, the partial-support options aren't enough - since as we don't have a `MediaStream`. Therefore, we would be supporting effectively only Safari with this feature)

Reply:
thanks